README.md

```diff
@@ -43,10 +43,28 @@ when building the images, e.g.:
 
 ## TileDB-R
 
-There is an image available for [TileDB-R](https://github.com/TileDB-Inc/TileDB-R)
-which is used only for CI. This image is build and designed to facilitate CI.
+There is two images available for [TileDB-R](https://github.com/TileDB-Inc/TileDB-R).
 
-### Instructions
+### User Image
+
+The user image is meant for end users to be able to use TileDB-R.
+
+#### Instructions
+
+1. Install the Docker daemon from https://www.docker.com/community-edition
```

Comment:
Simpler language:   "Install Docker from ..."    Windows or mac users don't need to know about a daemon service.

Reply:
I agree, I've removed this entire point (from all sections). Installing docker is outside the scope of this repo, the readme needs an overall.